### PR TITLE
Library project 

### PR DESCRIPTION
**Pull request by abhishek310001**
# Description:

In this PR I made use of Objects and its Constructor to that will make objects of book and stores it in an array. Initially it will contain 2 pre stored Books including their author pages and read status.

The screenshot of UI is attached in this PR for reference

## Feature:

- We can add a new Book using the Add Button at the top that will show up a form upon click.
- Upon successfully adding new book it will update the table containing list of Books and its Information.
- We can Remove a book by clicking on the delete icon on the Action column beside of each book.
- Also we can change the read status of a particular book by clicking on the Read/Unread button.

## UI Screenshot:

library-project

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+Book.prototype.changeReadStatus = function() {
+    if (this.readStatus === "Read") {
+        this.readStatus = "Unread";
+    }
+    else{
+        this.readStatus = "Read";
+    }
+}
+
+const theHobbits = new Book(
```
Comment (devanuj97) [inline script.js:22]: check if we can named value argument while creating object

File: script.js [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+  form.style.display = "none";
+}
+
+const displayBooks = () => {
+  if (myLibrary.length > 0) {
+      table.innerHTML = `
+      <tr class='table-heading'>
+      <th>Book Name</th>
+      <th>Author Name</th>
+      <th>Pages</th>
```
Comment (devanuj97) [inline script.js]: user indentation, for the sake of readability :)

File: script.js [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+    for (var i = 0; i < myLibrary.length; i++) {
+      rowEl = document.createElement("tr");
+      rowEl.innerHTML =`
+      <td>${myLibrary[i].title}</td>
+      <td><i>${myLibrary[i].author}</i></td>
+      <td>${myLibrary[i].pages}</td>
+      <td class="${myLibrary[i].readStatus}">${myLibrary[i].readStatus}</td>
+      <td>
+      <div class="btns">
+      <button class="rem-btn" onclick="removeBook(${i})"><img src="./icon/delete.svg" alt="Delete Icon"></button>
```
Comment (devanuj97) [inline script.js]: user indentation, for the sake of readability :)

**Review by devanuj97 — APPROVED** [review]
left some comment, please address before merge

**Comment by devanuj97** [comment]
Some more key pointers need to be fixed --
1. Form not resetting it's value after making `add book`

File: script.js (hunk 1 above)
Comment (abhishek310001) [reply script.js:22]: implemented the way of creating objects in which arguments are defined

File: script.js (hunk 2 above)
Comment (abhishek310001) [reply script.js]: fixed

File: script.js (hunk 3 above)
Comment (abhishek310001) [reply script.js]: fixed

**Pull request merged**